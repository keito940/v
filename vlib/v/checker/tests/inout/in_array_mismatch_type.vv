fn main() {
	if 1 in ['1', '2'] {
		println('ok')
	}
}