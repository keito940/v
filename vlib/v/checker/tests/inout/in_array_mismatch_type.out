vlib/v/checker/tests/inout/in_array_mismatch_type.v:2:7: error: the data type on the left of `in` does not match the array item type.
    1| fn main() {
    2|     if 1 in ['1', '2'] {
                ~~
    3|         println('ok')
    4|     }